src/components/Table.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import { connect } from 'react-redux';
+
+class Table extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.handleMove = this.handleMove.bind(this)
+    this.win = this.win.bind(this);
+  }
+
+  handleMove(index) {
+    const { whoMove, move, indexX, indexO, cells,
+      addIndexX, addIndexO, info, moveInfo, count } = this.props;
+
+    let copyCells = [...cells];
+    let copyIndexX = [...indexX];
+    let copyIndexO = [...indexO];
+
+    if (copyCells[index] !== null
+      || moveInfo === 'Победил: X'
+      || moveInfo === 'Победил: O'
```

Comment:
This is super-weak comparison. Create a variable like `winner` and use it instead. Don't use `UI` elements like `text string` in the code.